gopherpolicy/token.go

```diff
@@ -134,6 +134,10 @@ func (t *Token) ApplicationCredentialID() string {
 	return t.Context.Auth["application_credential_id"]
 }
 
+func (t *Token) IsAdminProject() string {
```

Comment:
1. Since this is a public method, please add a docstring. Suggestion:
   > IsAdminProject returns whether the token is scoped to the project that is designated for cloud administrators within Keystone (if any).
2. This should return `bool` instead of `string`. Add a private bool member on type Token if necessary.